### PR TITLE
fix: error on accessing nullable variables

### PR DESCRIPTION
**Pull request by ryo-gk**
Fix the error which occurs when `container` element is null.

### CODE REVIEW

**Review by kiaking — COMMENTED** [review]
Looks reasonable to me.

@brc-dd Could you take a look? 👀

File: lib/composables/Flyout.ts [hunk 1]
```diff
@@ -19,13 +19,13 @@ export function useFlyout(container?: Ref<any>) {
...
   function closeOnClickOutside(event: any) {
-    if (!el.value.contains(event.target) && isVisible(el.value)) {
+    if (!el.value?.contains(event.target) && isVisible(el.value)) {
       isOpen.value = false
     }
   }
 
   function isVisible(el: any) {
-    return !!el && !!(el.offsetWidth || el.offsetHeight || el.getClientRects().length)
+    return !!el && !!(el?.offsetWidth || el?.offsetHeight || el.getClientRects().length)
```
Comment (brc-dd) [inline lib/composables/Flyout.ts:28]: these might not be necessary since there is already a `!!el` check.

**Review by brc-dd — APPROVED** [review]

**Comment by brc-dd** [comment]
LGTM, but why aren't these handled by TS in the first place? (I see that everything in the flyout composable is `any`.)

**Comment by kiaking** [comment]
> LGTM, but why aren't these handled by TS in the first place? (I see that everything in the flyout composable is any.)

I'm pretty sure because we were too laze 😅

File: lib/composables/Flyout.ts (hunk 1 above)
Comment (ryo-gk) [reply lib/composables/Flyout.ts]: Thanks, you right!
I fixed it.

**Pull request merged**